### DISCUSSION TITLE
Welcome to vue-dynamic-height Discussions!

### DISCUSSION THREAD

**Discussion by guiooak** [guibrancopc/vue-dynamic-height#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guiooak** [comment]
Opa, teste de comentário.

**Comment by guiooak** [comment]
Outro comentário.

**Reply by guilherme-carneiro-ca** [reply]
bla

**Comment by guilherme-carneiro-ca** [comment]
topper

**Comment by cirocfc** [comment]
cool
